File: src/main/Python/boj_step15/boj4134.py
import sys
import math
cin = sys.stdin.readline

T = int(cin())
for _ in range(T):
    n = int(cin())
    while True:
        isPrime = True
        if n <= 1:
            print(2)
            break
        for i in range(2, int(math.sqrt(n)) + 1):
            if n % i == 0:
                isPrime = False
                break
        if isPrime:
            print(n)
            break
        else: n += 1

File: src/main/Cpp/boj_step15/boj4134.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main() {
    cin.tie(nullptr);
    ios_base::sync_with_stdio(false);

    int T;
    cin >> T;

    for (int i = 0; i < T; i++) {
        unsigned int n;
        cin >> n;

        if (n < 2) {
            cout << '2' << '\n';
            continue;
        }

        while (true) {
            bool isPrime = true;

            for (int j = 2; j <= sqrt(n); j++) {
                if (n % j == 0) {
                    isPrime = false;
                    break;
                }
            }

            if (isPrime) {
                cout << n << '\n';
                break;
            }

            n++;
        }
    }
}